src/applications/my-education-benefits/config/form.js

```diff
@@ -511,22 +511,28 @@ const formConfig = {
               'ui:description': (
                 <>
                   <h3>Review your phone numbers and email address</h3>
-                  <p>We’ll use this information to:</p>
+                  <p>
+                    <b>We’ll use this information to:</b>
```

Comment:
Note: `<strong>` would be more semantic here as `<b>` only indicates a styling difference, not a difference in importance.